### ISSUE TITLE
Student believes solution has incorrect code

### ISSUE THREAD

**Issue by lcrucks** [udacity/mlnd_issues_tracker#66]
Lesson 3: Deep Neural Networks, Concept 7 (Keras)

The network_solution.py script seems to have the incorrect code:

# Building the model 
xor = Sequential() 
xor.add(Dense(32, input_dim=2)) 
xor.add(Activation("sigmoid")) <------ This should be xor.add(Activation("tanh")) 
xor.add(Dense(2)) 
xor.add(Activation("sigmoid"))

*Set the first layer to a Dense() layer with an output width of 8 nodes and the input_dim set to the size of the training samples (in this case 2).

*Add a tanh activation function.

*Set the output layer width to 1, since the output has only two classes. (We can use 0 for one class an 1 for the other)

*Use a sigmoid activation function after the output layer. 
Run the model for 50 epochs.

**Comment by fixingitnow** [comment]
Could you give the classroom link to this as well? Thanks!
